src/modelfree/common/utils.py

```diff
@@ -2,6 +2,7 @@
 import datetime
 import itertools
 import os
+import shutil
```

Comment:
unused import

Reply:
I suggest running `ci/code_checks.sh` before commit which will catch this (and some other linting errors). I have it installed as a Git commit hook.